src/main/scala/org/ergoplatform/settings/Parameters.scala

```diff
@@ -0,0 +1,14 @@
+package org.ergoplatform.settings
+
+object Parameters {
+  // Max total computation cost of a block.
+  val MaxBlockCost: Long = 1000000
+
+  // Max size of transactions section of a block.
+  val MaxBlockSize: Int = 512 * 1024
+
+  // Cost of storing 1 byte per block, in nanoErgs
+  // with default value of 12 nanoErgs, storage cost for an (ordinary) output of 80 bytes would be ~1.01 Ergo per 4 years
+  // max should be about 24 probably
```

Comment:
should we have max at all? Why 24?

Reply:
As miner is charging "K*B*(out.value - self.value)" , but K here is taken at the moment of charging (not creation). Thus greedy miners have an incentive probably to rip short-term profits by increasing K (and limits in increasing the K are not seen).

Reply:
with K = 12, a box of 80 bytes per 4 years ~= 1 Ergo
K = 24 => ~= 2 Ergo